src/api/enums/EnvironmentType.ts

```diff
@@ -0,0 +1,5 @@
+export enum EnvironmentType {
+    DEVELOPMENT = 'DEVELOPMENT',
+    RELEASE = 'RELEASE',
+    ALL = 'ALL'
```

Comment:
can you change release to production, since its more commonly used name for the environment